spec.html

```diff
@@ -21095,29 +21129,31 @@ <h1>Runtime Semantics: Evaluation</h1>
         </emu-alg>
         <emu-grammar>LexicalBinding : BindingIdentifier</emu-grammar>
         <emu-alg>
-          1. Let _lhs_ be ResolveBinding(StringValue of |BindingIdentifier|).
-          1. Return InitializeReferencedBinding(_lhs_, *undefined*).
+          1. Let _lhs_ be Completion(ResolveBinding(StringValue of |BindingIdentifier|)).
+          1. Perform ? InitializeReferencedBinding(_lhs_, *undefined*).
+          1. Return ~empty~.
         </emu-alg>
         <emu-note>
           <p>A static semantics rule ensures that this form of |LexicalBinding| never occurs in a `const` declaration.</p>
         </emu-note>
         <emu-grammar>LexicalBinding : BindingIdentifier Initializer</emu-grammar>
         <emu-alg>
           1. Let _bindingId_ be StringValue of |BindingIdentifier|.
-          1. Let _lhs_ be ResolveBinding(_bindingId_).
+          1. <!-- TODO: Figure out if ResolveBinding can actually throw here. --> Let _lhs_ be Completion(ResolveBinding(_bindingId_)).
```

Comment:
Delete or resolve TODO comment?

Reply:
Deleted, but I think we should still try to figure this out.